### ISSUE TITLE
NeurIPS2018 train test not matching

### ISSUE THREAD

**Issue by charliedream1** [AI4Finance-Foundation/FinRL-Tutorials#50]
turbulence_threshold and risk_indicator_col are used during backtest, however, not used in training, will it causing unmatching problem?

**Comment by XiaoYangLiu-FinRL** [comment]
We believe that a trader may NOT direct follow the agent's trading signal, instead, it may perform some kind of risk control.  
An agent may not be able to deal with any kind of market situation, like a market crush, that has not been seen in the training data.  It depends on your settings.

**Comment by charliedream1** [comment]
thx，I see now，so we still need to combine model with man-made rules to make a better strategy



---Original---
From: "Xiao-Yang ***@***.***&gt;
Date: Mon, Mar 27, 2023 12:50 PM
To: ***@***.***&gt;;
Cc: "Optimus ***@***.******@***.***&gt;;
Subject: Re: [AI4Finance-Foundation/FinRL-Tutorials] NeurIPS2018 train testnot matching (Issue #50)




 
We believe that a trader may NOT direct follow the agent's trading signal, instead, it may perform some kind of risk control.
 An agent may not be able to deal with any kind of market situation, like a market crush, that has not been seen in the training data.  It depends on your settings.

**Comment by XiaoYangLiu-FinRL** [comment]
> thx，I see now，so we still need to combine model with man-made rules to make a better strategy
> […](#)
> ---Original--- From: "Xiao-Yang ***@***.***&gt; Date: Mon, Mar 27, 2023 12:50 PM To: ***@***.***&gt;; Cc: "Optimus ***@***.******@***.***&gt;; Subject: Re: [AI4Finance-Foundation/FinRL-Tutorials] NeurIPS2018 train testnot matching (Issue #50) We believe that a trader may NOT direct follow the agent's trading signal, instead, it may perform some kind of risk control. An agent may not be able to deal with any kind of market situation, like a market crush, that has not been seen in the training data. It depends on your settings. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

Yes.  Deep learning and reinforcement learning may provide new methods that do not overlap with existing strategies, worth exploring.

**Comment by charliedream1** [comment]
thx for reply



---Original---
From: "Xiao-Yang ***@***.***&gt;
Date: Mon, Mar 27, 2023 12:58 PM
To: ***@***.***&gt;;
Cc: "Optimus ***@***.******@***.***&gt;;
Subject: Re: [AI4Finance-Foundation/FinRL-Tutorials] NeurIPS2018 train testnot matching (Issue #50)




  
thx，I see now，so we still need to combine model with man-made rules to make a better strategy
 …
 ---Original--- From: "Xiao-Yang @.&gt; Date: Mon, Mar 27, 2023 12:50 PM To: @.&gt;; Cc: "Optimus @.@.&gt;; Subject: Re: [AI4Finance-Foundation/FinRL-Tutorials] NeurIPS2018 train testnot matching (Issue #50) We believe that a trader may NOT direct follow the agent's trading signal, instead, it may perform some kind of risk control. An agent may not be able to deal with any kind of market situation, like a market crush, that has not been seen in the training data. It depends on your settings. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.***&gt;
  
Yes.  Deep learning and reinforcement learning may provide new methods that do not overlap with existing strategies, worth exploring.
